src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -372,35 +375,28 @@ private static ICFamilyRulesConfig GetDummyRulesConfiguration()
             return config;
         }
 
-        private Request GetSuccessfulRequest(IAnalyzerOptions analyzerOptions)
+        private Request GetSuccessfulRequest(IAnalyzerOptions analyzerOptions = null,
```

Comment:
Modified the helper method to use the normal pattern for optional test inputs (the new property-checking tests above need to supply specific instances).